connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java

```diff
@@ -283,24 +283,27 @@ private void doCommitSync(Map<TopicPartition, OffsetAndMetadata> offsets, int se
         }
     }
 
+    private void doCommitAsync(Map<TopicPartition, OffsetAndMetadata> offsets, final int seqno) {
+        log.info("{} Committing offsets asynchronously using sequence number: {}", this, seqno, offsets);
+        OffsetCommitCallback cb = new OffsetCommitCallback() {
+            @Override
+            public void onComplete(Map<TopicPartition, OffsetAndMetadata> offsets, Exception error) {
+                log.debug("{} Complete asynchronous offset commit for sequence number {}: {}", this, seqno, offsets);
```

Comment:
Is this log line needed? Seems like we get all this info in `onCommitCompleted`.